db/db.js

```diff
@@ -379,6 +379,44 @@ async function getLocationData(locationid) {
 
 }
 
+
+//Retrieves the partcle door coreID corresponding to a locationid
+//This is required for ServerTest.js
+
+async function getDoorParticleCoreIDFromLocationID(locationid){
```

Comment:
Where is this used?

Reply:
I think this is no longer used - it used to be part of the mock im21Door function - it would take a locationid as a parameter and then use that to find a doorCoreID to include in the post request, but I took that out in the last changes.